src/View/OutputView.js

```diff
@@ -0,0 +1,25 @@
+const { Console } = require('@woowacourse/mission-utils');
+const { MESSAGE, RESULT } = require('../Util/Constants');
+
+const OutputView = {
+  printStart() {
+    Console.print(MESSAGE.START);
+  },
+
+  printResult({ strike, ball }) {
+    const strikeResult = strike ? strike + RESULT.STRIKE : '';
+    const ballResult = ball ? ball + RESULT.BALL + RESULT.BLANK : '';
```

Comment:
Airbnb의 코드컨벤션에선 String을 만들 때에는 +보단 Template string을 사용하기를 권장하고 있습니다.
https://github.com/airbnb/javascript#es6-template-literals

```suggestion
    const strikeResult = strike ? `${strike}${RESULT.STRIKE}` : '';
    const ballResult = ball ? `${ball}${RESULT.BALL}${RESULT.BLANK} : '';
```